.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
I'd clarify these are example reproductions, theirs might not be the same

Reply:
yeah this basically the default template, and it assumes the project is web based. Not entirely sure what *exactly* we should have in these, got any ideas?

Reply:
I can't come up with anything good...I like the rest of the template (even if it is generic) but these steps are too much IMO. Maybe just remove the steps and say "List the steps to reproduce to the behavior."